packages/core/src/codewhisperer/util/zipUtil.ts

```diff
@@ -5,11 +5,13 @@
 import admZip from 'adm-zip'
 import * as vscode from 'vscode'
 import path from 'path'
-import { tempDirPath } from '../../shared/filesystemUtilities'
+import { readFileAsString, tempDirPath } from '../../shared/filesystemUtilities'
 import { getLogger } from '../../shared/logger'
 import * as CodeWhispererConstants from '../models/constants'
+import { existsSync, readdirSync, statSync } from 'fs'
```

Comment:
Going forward we are avoiding the use of `fs` and `fs-extra` for the extension to be compatible in the browser.

Please look at `FileSystemCommon` and start using that instead. It should have what you need.